packages/react-data-grid/src/EmptyChildRow.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+const EmptyChildRow = ({treeDepth, cellHeight, onAddSubRow, AddNewText = 'Add New'}) => {
+  const height = 12;
+  const width = 12;
+  let left = treeDepth * 15;
+  let top = (cellHeight - 12) / 2;
+  let style = {
+    height: cellHeight,
+    borderBottom: '1px solid #dddddd'
+  };
+  return (<div className="react-grid-Row" style={style}>
+    <div className="react-grid-Cell" style={{ position: 'absolute', height: cellHeight, width: '100%' }}>
+      <div className="rdg-empty-child-row" style={{ marginLeft: '30px', lineHeight: `${cellHeight}px` }}>
+      <div className="'rdg-child-row-action-cross rdg-child-row-action-cross-last" />
```

Comment:
@malonecj noticed an apostrophe here in front of  **rdg-child-row-action-cross rdg-child-row-action-cross-last**

was that intentional?